syntax = "proto2";
package units.proto;

import "util/proto/object_id.proto";
import "games/market/proto/goods.proto";
import "games/population/proto/consumption.proto";

message Mobility {
  optional uint64 speed_u = 1;
  optional uint64 max_weight_u = 2;
  optional uint64 max_bulk_u = 3;
}

message Template {
  optional Mobility mobility = 1;
  optional util.proto.ObjectId template_id = 2;
  optional uint64 base_action_points_u = 3;
  repeated string tags = 4;
  optional market.proto.Container attrition = 5;
  repeated consumption.proto.Consumption supplies = 6;
}
